src/client/components/board/board.js

```diff
@@ -101,7 +102,8 @@ class Board extends React.Component {
           </div>
         </div>
         <Sidebar playerID={this.props.playerID} matchID={this.props.matchID} G={this.props.G} ctx={this.props.ctx} moves={this.props.moves} isInThreatStage={isInThreatStage} current={current} active={active} names={this.state.names} />
-        <Threatbar gameMode={this.props.G.gameMode} suit={this.props.G.suit} playerID={this.props.playerID} model={this.state.model} names={this.state.names} G={this.props.G} ctx={this.props.ctx} moves={this.props.moves} active={active} isInThreatStage={isInThreatStage} />
+        <Threatbar playerID={this.props.playerID} model={this.state.model} names={this.state.names} G={this.props.G} ctx={this.props.ctx} moves={this.props.moves} active={active} isInThreatStage={isInThreatStage} />
+        <ThreatModal isOpen={this.props.G.threat.modal} G={this.props.G} ctx={this.props.ctx} playerID={this.props.playerID} moves={this.props.moves} names={this.state.names} />
```

Comment:
I believe to me it would make more sense to have `ThreatModal` as a child of `Threatbar`, rather than a sibling.

Reply:
I think this makes sense, I'll move it.